resources/components/ClientsTable.vue

```diff
@@ -2,13 +2,39 @@
     <div>
         <form class="d-flex justify-content-end" method="get" action="">
             <div class="form-inline mb-2">
+                <div class="form-check-inline">
```

Comment:
Jw bo to wygląda jak powielenie kodu tylko zmieniają się labele i input Id name na bazie tego typu.